app/build.gradle.kts

```diff
@@ -114,6 +117,7 @@ dependencies {
     implementation(libs.androidx.lifecycle.liveData)
     implementation(libs.androidx.lifecycle.runtime)
     implementation(libs.androidx.lifecycle.viewModelSavedState)
+    implementation("androidx.fragment:fragment-ktx:1.4.1")
```

Comment:
Please Ignore. This is removed in part 2.
This was part of the prev spike work.